src/main/java/uk/gov/hmcts/reform/logging/exception/AlertLevel.java

```diff
@@ -0,0 +1,5 @@
+package uk.gov.hmcts.reform.logging.exception;
+
+public enum AlertLevel {
+    P1, P2, P3, P4;
+}
```

Comment:
Worth adding javadoc comment on this enum imho?

Why do values start with 'P'?